src/components/Tabs/Tabs/Tab.tsx

```diff
@@ -0,0 +1,20 @@
+import { useParams } from 'react-router-dom';
+import { TabType } from '../../../types/Tab';
+
+interface Props {
+  tabs: TabType[],
+}
+
+export const Tab:React.FC<Props> = ({
+  tabs,
+}) => {
+  const { id } = useParams();
+
+  return (
+    <>
+      {tabs.map((tab: TabType) => (
+        tab.id === id ? <p key={tab.id}>{tab.content}</p> : null
+      ))}
+    </>
```

Comment:
```suggestion

      {tabs.find((tab: TabType) => (
        tab.id === id
      ))}

```
Consider using the find function here instead of the map and showing tab content of that tab here